.github/actions/bootstrap/action.yaml

```diff
@@ -0,0 +1,99 @@
+name: "Bootstrap"
```

Comment:
This uses the "Bootstrap" action that we also have built in syft. Note: this adds a few other actions including configuring python tooling an caches.